docs/cli.md

```diff
@@ -133,13 +138,24 @@ To generate a TailCall GraphQL configuration, provide a configuration file to th
 
 The `inputs` section specifies the sources from which the GraphQL configuration should be generated. Each source can be either a REST endpoint or a protobuf file.
 
-1. **REST:** For REST endpoints, provide the endpoint URL (`src`) and the field name in the configuration (`fieldName`), which will be used as the field name in the operation type.
+1. **REST:** When defining REST endpoints, the configuration should include the endpoint URL (src), the required headers for making the HTTP request, and the field name (fieldName) that will be used as the field name in the operation type.
+
+   1. **Headers:** Users can specify the required headers to make the HTTP request in the headers section.
```

Comment:
Between different fields — src, fieldName, headers - We should specify what's required and what's optional and what are the default values.